### ISSUE TITLE
An extra question mark is added when doing nlp-parse?

### ISSUE THREAD

**Issue by leungmanhin** [opencog/opencog#2375]
When I do `(nlp-parse "what about landru")`, there is a question mark appends to the end of the word "landru" even if there is no question mark in the input sentence. For example the `PredicateNode` as in the R2L output:
```
(ReferenceLink
   (InterpretationNode "sentence@a6f4e7ef-64bf-4759-aada-7f0abdf7b427_parse_0_interpretation_$X")
   (SetLink
      (ImplicationLink
         (PredicateNode "landru?@2c72b23d-5a18-4697-a75f-979a197edcbb")
         (PredicateNode "landru?" (stv 0.33333334 0.0012484394))
      )
      (InheritanceLink
         (ConceptNode "what@47927ae0-89dd-4d18-b0b7-18187cbdc00a")
         (ConceptNode "what" (stv 0.16666667 0.0012484394))
      )
      (EvaluationLink
         (PredicateNode "landru?@2c72b23d-5a18-4697-a75f-979a197edcbb")
         (ListLink
            (ConceptNode "what@47927ae0-89dd-4d18-b0b7-18187cbdc00a")
         )
      )
      (InheritanceLink
         (InterpretationNode "sentence@a6f4e7ef-64bf-4759-aada-7f0abdf7b427_parse_0_interpretation_$X")
         (DefinedLinguisticConceptNode "InterrogativeSpeechAct")
      )
      (InheritanceLink
         (PredicateNode "landru?@2c72b23d-5a18-4697-a75f-979a197edcbb")
         (DefinedLinguisticConceptNode "present")
      )
   )
)
```

or the `WordNode`:
```
(ReferenceLink (stv 1 1)
   (WordInstanceNode "landru?@2c72b23d-5a18-4697-a75f-979a197edcbb")
   (WordNode "landru?" (stv 0.090909094 0.0012484394))
)
```

**Comment by linas** [comment]
This is another relex bug.  link-grammar spits out [?] to indicate that it did not know that word, but it guessed it.  Then relex removes the square brackets ...

**Comment by linas** [comment]
fixed here: opencog/relex@84e4edef77ed3c3f1a588f47f814cc7719832fad just now

**Comment by leungmanhin** [comment]
thanks!!
